janusgraph-hbase-parent/janusgraph-hbase-core/src/test/java/org/janusgraph/diskstorage/hbase/HBaseStoreManagerMutationTest.java

```diff
@@ -127,4 +129,33 @@ public void testKCVMutationToPuts() throws Exception {
         Collections.sort(deleteColumns);
         Assert.assertArrayEquals(expectedColumnDelete.toArray(), deleteColumns.toArray());
     }
+
+    @Test
+    public void testMutationToPutsTTL() throws Exception{
+        StaticBuffer col, val;
+        StaticArrayEntry e;
+        final List<Long> expectedColumnsWithTTL = new ArrayList<>();
+        final List<Long> putColumnsWithTTL = new ArrayList<>();
+        int row = 1;
```

Comment:
Redundant variable

Reply:
Nice, Codacy [agrees with you](https://github.com/JanusGraph/janusgraph/pull/1456/files#r261649854) 😃

Reply:
Codacy works, Hurray! 😃